packages/steiger-plugin-fsd/src/no-cross-imports/README.md

```diff
@@ -0,0 +1,108 @@
+# `no-cross-imports`
+
+This rule forbids cross-imports between slices on the same layer. This is in accordance to the slice isolation principle in Feature-Sliced Design:
+
+> A module in a slice can only import other slices when they are located on layers strictly below.
+>
+> https://feature-sliced.design/docs/reference/layers#import-rule-on-layers
+
+Example of a project structure that passes this rule (arrows signify imports):
+
+```mermaid
+flowchart BT
+  subgraph entities
+    subgraph entities/user[user]
+      subgraph entities/user/ui[ui]
+        entities/user/ui/UserAvatar.tsx[UserAvatar.tsx]
+      end
+      entities/user/index.ts[index.ts]
+    end
+    subgraph entities/product[product]
+      subgraph entities/product/ui[ui]
+        entities/product/ui/ProductCard.tsx[ProductCard.tsx]
+      end
+      entities/product/index.ts[index.ts]
+    end
+  end
+
+  subgraph shared
+    subgraph shared/ui[ui]
+      shared/ui/index.ts[index.ts]
+    end
+  end
+
+  entities/user/ui/UserAvatar.tsx --> shared/ui/index.ts
+  entities/product/ui/ProductCard.tsx --> shared/ui/index.ts
+```
+
+```mermaid
+flowchart BT
+  subgraph entities
+    subgraph entities/user[user]
+      subgraph entities/user/@x[@x]
```

Comment:
issue: `@` is not a valid symbol in a Mermaid diagram node name, which causes an error on GitHub preview

suggestion: use `at-` as a prefix to overcome this limitation. The label written in square brackets can still use `@`:
```suggestion
      subgraph entities/user/at-x[@x]
```